scRNA-seq/02-tag-based_pre-processing_scRNA-seq.md

```diff
@@ -139,13 +139,11 @@ This will provide html output with graphs evaluating the data.
 ```r
 # Produce a QC report
 alevinQC::alevinQCReport(alevin_file,
-                         sampleId = "pbmc_1k_v2_S1_L001_subset",
-                         outputFile = "pbmc_1k_v2_S1_L001_qc_report.html",
+                         sampleId = "tabula_muris_P4_3_S1_L001_R1_001",
+                         outputFile = "tabula_muris_P4_3_S1_L001_R1_001_qc_report.html",
                          outputDir = "data",
                          outputFormat = "html_document")
 ```
-Now you can check out "pbmc_1k_v2_S1_L001_qc_report.html" in order to examine
+Now you can check out "tabula_muris_P3_S1_L001_R1_001_qc_report.html" in order to examine
 the quality of your data and performance of Alevin.
 
-Here's the full pbmc dataset to compare to:
```

Comment:
You now have a _bad_ example from the liver samples correct?

Reply:
Yes, however, I've recently realized what a huge idiot I am and that I need to re-quantify the Tabula Muris data, because I never switched out the human transcriptome index for the mouse one. :( It's bound to look much better after I use the right organism. :( Shame.